lib/paper_trail/config.rb

```diff
@@ -21,7 +21,8 @@ class Config
       :association_reify_error_behaviour,
       :object_changes_adapter,
       :serializer,
-      :version_limit
+      :version_limit,
+      :has_paper_trail_defaults
```

Comment:
Instead of an `attr_accessor`, should we synchronize using `@mutex` as we do with `#enabled=`?

Reply:
Hmm - while I get it, that means mutex overhead for every model class load to reference it.  My gut says this is a config-time thing and an appropriate comment saying this isn't thread safe and intended for initialization time use would avoid adding synchronization overhead.  Given the likely usage pattern (e.g. initializer one-shot stuff), I would avoid it, but if you feel differently, it's not much work.